### ISSUE TITLE
Show full path to project in Gradle Tasks view

### ISSUE THREAD

**Issue by donat** [eclipse/buildship#632]
[Report](https://bugs.eclipse.org/bugs/show_bug.cgi?id=529350) from Scott Warner:

> Can you show the folders in the Gradle Tasks view as the user would type them in a console?  
> 
> i.e.
> :Project1:API
> :Project1:Service
> :Project2:API
> :Project2:Service
> 
> It would make it easier to distinguish between sub-projects and would be a hint to the user what they would need to type in a console to run a task.

**Comment by donat** [comment]
I generally like the idea as it would also make it easier to filter on the tasks. Let's see how many people are going to 👍 this issue or if there's somebody in the community wants to implement it. In the latter case, we're happy to help get started.

**Comment by Camork** [comment]
Hi,
   I am glad to implement this feature, and i thought it should be like a hierarchy structure view before. Is this better than just project name in each project nodes? This is my idea.
![image](https://user-images.githubusercontent.com/19710731/39683295-4b40c6d0-51e7-11e8-80c6-47cce8ab311c.png)

**Comment by nniesen** [comment]
I preferred the flat alphabetical Gradle STS task list. It was quicker expanding trees. Two sorely missed features are the filter and the "run Gradle command" where I specify multiple tasks with our without the "project" selector (":"). Would be nice if users could choose between a flat view within a project or a hierarchical view.

**Comment by donat** [comment]
@Camork your idea can be added to Buildship as a new `Order by` option for the context menu.

**Comment by donat** [comment]
The project is always open for contribution. We are happy to guide anyone through the process.

**Comment by Camork** [comment]
Yeah, It should be an option to user as we don't know what kind of view user wanted.
